src/index.css

```diff
@@ -2,6 +2,8 @@
 
 body {
   --primary-color: #00cc9b;
+  --emphasis-color: #FA504F;
```

Comment:
It's usually named `accent color`

Ref:
- accent color: https://www.thespruce.com/what-are-accent-colors-5218695

Reply:
updated, learned a lot here, I thought accent color was a set of color pallet.